lua/nvim-tree/actions/node/open-file.lua

```diff
@@ -209,7 +209,15 @@ local function open_in_new_window(filename, mode, win_ids)
     cmd = string.format("edit %s", fname)
   end
 
-  set_current_win_no_autocmd(target_winid)
+  if mode == "preview" then
```

Comment:
This is correct, however is only desirable for floating windows. There may be (future) WinLeave events in the non-floating case that we still wish to receive.

- [ ] Please change to `if mode == "preview" and view float is set then`